src/xma/src/xmaapi/xma_utils.cpp

```diff
@@ -693,6 +694,28 @@ int32_t check_all_execbo(XmaSession s_handle) {
     return XMA_SUCCESS;
 }
 
+int32_t xma_check_device_buffer(const XmaBufferObj* b_obj) {
+    if (!b_obj) {
+        xma_logmsg(XMA_ERROR_LOG, XMAUTILS_MOD, "xma_check_device_buffer failed. XMABufferObj failed allocation\n");
+        return XMA_ERROR;
+    }
+
+    auto b_obj_priv = reinterpret_cast<XmaBufferObjPrivate*>(b_obj->private_do_not_touch);
```

Comment:
Will this compile?  I think it should be` const XmaBufferObjPrivate*` now that argument correctly is const.

Reply:
not seeing any compilation issue.